### DISCUSSION TITLE
The script is in the Arch User Repository

### DISCUSSION THREAD

**Discussion by gianmarcogg03** [ThioJoe/YT-Spammer-Purge#775] [Ideas]
This script was added to the AUR a few months ago, so it would be a good idea to add a paragraph in the Linux installation instructions saying that Arch users can install it from the AUR with something like `yay -S yt-spammer-purge`.
The package webpage is [here](https://aur.archlinux.org/packages/yt-spammer-purge).

**Comment by txtsd** [comment]
No, before a system package of this sort can be endorsed, yt-spammer-purge first needs to adhere to the freedesktop [XDG Base Directory Specification](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html).

I created a discussion about that a few days ago at  #741.

**Reply by KendallDoesCoding** [reply]
You should do this @ThioJoe !

**Reply by ThioJoe** [reply]
I'd be lying if I said there was a realistic chance of me doing this

**Reply by KendallDoesCoding** [reply]
> I'd be lying if I said there was a realistic chance of me doing this

ok!

**Reply by txtsd** [reply]
@ThioJoe If I did a PR, would you accept it though?
